compositionalityarticle.cls

```diff
@@ -927,7 +927,7 @@ class for Compositionality - the open journal for the mathematics of composition
 {\fancyfoot[R]{\sffamily\iftoggle{@unpublished}{\thepage}{\textcolor{compositionalitypink}{\thepage}}}}
 {\fancyfoot[C]{\sffamily\iftoggle{@unpublished}{\thepage}{\textcolor{compositionalitypink}{\thepage}}}}
 \iftoggle{@accepted}
-	{\fancyfoot[L]{\usebox{\@compositionalityacceptedbox}}
```

Comment:
This way `fancyfoot` acts only on the first page, and repetition is avoided; I think haing a "watermark" ensuring the paper is legit _on each page_ is a bit redundant, ugly, and distracts the reader.